hiss/hiss/settings/customization.py

```diff
@@ -5,5 +5,6 @@
 EVENT_START_DATETIME = timezone.datetime(2020, 1, 25, hour=9, minute=0, second=0)
 EVENT_END_DATETIME = timezone.datetime(2020, 1, 26, hour=12, minute=0, second=0)
 LOGIN_REDIRECT_URL = "/status/"
+LOGOUT_REDIRECT_URL = "/accounts/login/"
```

Comment:
Could you change these to be `reverse_lazy`?

Also, we [already have a reference to `LOGOUT_REDIRECT_URL`](https://github.com/tamuhack-org/Ouroboros/blob/800e9a93757f1ed2d04c0b09b4f084c1979e6d92/hiss/hiss/settings/base.py#L85), I think we should move both of these settings into `base.py`.